pkg/apis/internal.acorn.io/v1/appinstance.go

```diff
@@ -159,10 +163,23 @@ type AppInstanceStatus struct {
 	AvailableAppImage      string                     `json:"availableAppImage,omitempty"`
 	ConfirmUpgradeAppImage string                     `json:"confirmUpgradeAppImage,omitempty"`
 	AppSpec                AppSpec                    `json:"appSpec,omitempty"`
+	Defaults               Defaults                   `json:"defaults,omitempty"`
+	Scheduling             map[string]Scheduling      `json:"scheduling,omitempty"`
```

Comment:
Why does this need to be on the `Status`? Can't we calculate this on the fly when creating the Kubernetes containers?

Reply:
These are on the status so that if the WorkloadClass changes we don't recalculate. So we calculate it only when an app gets updated.